doc/CMakeLists.txt

```diff
@@ -6,27 +6,29 @@ project(Zephyr-Kernel-Doc LANGUAGES)
 set(NO_BOILERPLATE TRUE)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE} ..)
 
+#-------------------------------------------------------------------------------
+# Dependencies
+
+set(DOXYGEN_SKIP_DOT True)
+find_package(Doxygen REQUIRED)
+
+find_program(SPHINXBUILD sphinx-build)
+if(NOT SPHINXBUILD)
+  message(FATAL_ERROR "The 'sphinx-build' command was not found")
+endif()
+
 find_program(WEST west)
 if(NOT WEST)
   message(FATAL_ERROR "The 'west' command was not found")
 endif()
 
-file(TO_CMAKE_PATH "${ZEPHYR_BASE}" ZEPHYR_BASE)
-
-message(STATUS "Zephyr base: ${ZEPHYR_BASE}")
```

Comment:
Why is this being removed ?

Reply:
I can add the status back, missed that one. Regarding `file(TO_CMAKE_PATH "${ZEPHYR_BASE}" ZEPHYR_BASE)`, is it really needed?

Reply:
even with your PR then you are still using ZEPHYR_BASE several places:
https://github.com/teslabs/zephyr/blob/6cbfbb14d613cc358c4b13224acb9283e59b3398/doc/CMakeLists.txt#L45
https://github.com/teslabs/zephyr/blob/6cbfbb14d613cc358c4b13224acb9283e59b3398/doc/CMakeLists.txt#L150-L155
https://github.com/teslabs/zephyr/blob/6cbfbb14d613cc358c4b13224acb9283e59b3398/doc/CMakeLists.txt#L173-L174

and having a risk that paths are suddenly having mixed `\` and `/` just because someone tries to build the docs on Windows is a risk I would not personally take.

Reply:
added back